bulk_data_gen/metaqueries/generate_data.go

```diff
@@ -0,0 +1,115 @@
+package metaqueries
+
+import (
+	"time"
+
+	"math/rand"
+
+	"github.com/influxdata/influxdb-comparisons/bulk_data_gen/common"
+)
+
+// letters are used for generating the pseudo-random tag values used in this
+// dataset. Currently only the first 10 letters are used.
+const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Just out of curiosity, why not use something like a truncated [Type 4 UUID](https://pkg.go.dev/github.com/google/uuid#NewRandom) instead?

Reply:
Good idea! I'll incorporate that. Looks like it makes the data generation slightly more efficient.